examples/all-options.xml

```diff
@@ -184,7 +184,8 @@
           </basis>
           <origin> <x>0.0</x> <y>0.0</y> <z>0.0</z> </origin>
         </generator>
-
+        
+        <ignoreCheckpointTime>true</ignoreCheckpointTime> <!-- If true, the simulation ignores the timestep in the loaded checkpoint and starts the simulation from timestep 0. Default: false-->
```

Comment:
This looks like a new a general new I/O feature and not like a fix related to MaMiCo. I suggest merging seperately with additional documentation.

Reply:
This fix was originally part of #232 and wasn't reflected in `all-options.xml`, so I put it in here. I'll make it a separate PR.